docs/services/aws-services-guide.md

```diff
@@ -895,28 +459,20 @@ In this guide we:
 * Configured RDS to communicate with EKS
```

Comment:
Should we keep the fact that we setup an EKS cluster in this list?

Reply:
Nope, good catch!